### PR TITLE
Use PORT=8080 as default env

### LINKED ISSUE #430: Use a default port if none is given
## Describe the Enhancement

Currently, if users don't set the `PORT` environment variable when running the application image, they can get an error message that is hard to understand: `host not found in "default_server" of the "listen" directive`. The primary issue is that the `{{port}}` interpolation into the template results in an empty field which mangles the directive.

## Possible Solution

If the user has not set the `PORT` environment variable, the `{{port}}` template function should use a default value like `8080`.

## Motivation

Doing this would help to streamline the experience for users of the buildpack and reduce the number of environment variables required to run applications using this buildpack.

### PR DESCRIPTION
**Pull request by c0d1ngm0nk3y**
fixes #430 

## Summary

The environment variable `PORT` is set to `8080`, so that nginx starts by default on this port. If the user does not overwrite it.

## Use Cases

> Doing this would help to streamline the experience for users of the buildpack and reduce the number of environment variables required to run applications using this buildpack.

## Checklist

* [X] I have viewed, signed, and submitted the Contributor License Agreement.
* [X] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [X] I have added an integration test, if necessary.
* [X] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [X] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

**Comment by ForestEckhardt** [comment]
Other than the above comment I think that this all looks good.

File: integration/simple_app_test.go [hunk 1]
```diff
@@ -92,6 +92,42 @@ func testSimpleApp(t *testing.T, context spec.G, it spec.S) {
...
+
+			container, err = docker.Container.Run.
+				WithEnv(map[string]string{"PORT": "8081"}).
+				WithPublish("8081").
+				Execute(image.ID)
+			Expect(err).ToNot(HaveOccurred())
+
+			Eventually(container).Should(Serve(ContainSubstring("Hello World!")).WithEndpoint("/index.html"))
+		})
+
```
[the remark this answers is not in the document]
Comment (c0d1ngm0nk3y) [reply integration/simple_app_test.go:130]: The reason, I did add another test was that `BP_WEB_SERVER` has to be set in order to use the default port (line 101). What should be the behaviour without `BP_WEB_SERVER`?

[the remark this answers is not in the document]
Comment (ForestEckhardt) [reply integration/simple_app_test.go:130]: Oh I see. That is fair enough then.

**Review by ForestEckhardt — APPROVED** [review]

**Comment by ForestEckhardt** [comment]
@c0d1ngm0nk3y If you could just rebase this branch then I think we would be good to go!

**Comment by ForestEckhardt** [comment]
Bump @c0d1ngm0nk3y

**Comment by c0d1ngm0nk3y** [comment]
> Bump @c0d1ngm0nk3y

Sorry for the delay...

**Pull request merged**